src/kOS/Control/WheelSteeringManager.cs

```diff
@@ -0,0 +1,160 @@
+﻿using kOS.Safe.Encapsulation;
+using kOS.Safe.Exceptions;
+using kOS.Safe.Utilities;
+using kOS.Suffixed;
+using kOS.Utilities;
+using System;
+using UnityEngine;
+
+namespace kOS.Control
+{
+    internal class WheelSteeringManager : IFlightControlParameter
+    {
+        private Vessel internalVessel;
+        private uint controlPartId;
+        private SharedObjects controlShared;
+
+        public bool Enabled { get; private set; }
+        public float Value { get; set; }
+
+        public WheelSteeringManager(Vessel vessel)
+        {
+            Enabled = false;
+            controlPartId = 0;
+
+            internalVessel = vessel;
+        }
+
+        uint IFlightControlParameter.ControlPartId
+        {
+            get
+            {
+                return controlPartId;
+            }
+        }
+
+        bool IFlightControlParameter.Enabled
+        {
+            get
+            {
+                return Enabled;
+            }
+        }
+
+        bool IFlightControlParameter.IsAutopilot
+        {
+            get
+            {
+                return true;
+            }
+        }
+
+        void IFlightControlParameter.CopyFrom(IFlightControlParameter origin)
+        {
+            object val = origin.GetValue();
+            Value = Convert.ToSingle(val);
+        }
+
+        void IFlightControlParameter.DisableControl()
+        {
+            controlShared = null;
+            controlPartId = 0;
+            Enabled = false;
+        }
+
+        void IFlightControlParameter.DisableControl(SharedObjects shared)
+        {
+            if (shared.KSPPart.flightID == controlPartId)
+                return;
+            ((IFlightControlParameter)this).DisableControl();
+        }
+
+        void IFlightControlParameter.EnableControl(SharedObjects shared)
+        {
+            controlPartId = shared.KSPPart.flightID;
+            controlShared = shared;
+            Enabled = true;
+        }
+
+        SharedObjects IFlightControlParameter.GetShared()
+        {
+            return controlShared;
+        }
+
+        object IFlightControlParameter.GetValue()
+        {
+            if (Enabled)
+            {
+                return Value;
+            }
+            return internalVessel.ctrlState.mainThrottle;
+        }
+
+        void IFlightControlParameter.UpdateAutopilot(FlightCtrlState c)
+        {
+            if (!Enabled) return;
+
+            if (!(controlShared.Vessel.horizontalSrfSpeed > 0.1f)) return;
+
+            if (Mathf.Abs(VesselUtils.AngleDelta(VesselUtils.GetHeading(controlShared.Vessel), VesselUtils.GetVelocityHeading(controlShared.Vessel))) <= 90)
+            {
+                c.wheelSteer = Mathf.Clamp(Value / -10, -1, 1);
```

Comment:
I never noticed this before in our code (and it was true before your PR, I checked.  You didn't change this.)  We divide the value by 10 (well, -10 technically, so we not only divide it, we invert the sign).  so the wheelsteer in the control is 1/10th as big as the one the script gives.  To get a real wheelsteer of 1, does this mean the script needs to set it to 10, rather than setting it to just 1.0?  Was there some reason we did this that I don't follow?  Wheelsteering appears to work just fine in practice, and doesn't feel like it's nerfed to only 1/10th deflection.

Is there some strange arcane rule in the KSP API that wheelSteer doesn't range from -1.0 to 1.0 like all the the other controls in FlightControlState do?